makefile

```diff
@@ -11,7 +11,9 @@ ifneq "$(SUPPORTS_MAKE_ARGS)" ""
 endif
 
 build:
-	docker-compose run node node_modules/babel-cli/bin/babel.js lib -d build --blacklist regenerator
+	docker-compose run node node_modules/babel-cli/bin/babel.js lib -d build --blacklist regenerator --ignore *.spec.js
```

Comment:
what is the effect of blacklist ? It looks like it has been removed in babel 6 (https://github.com/babel/babel/issues/2951)